src/components/expanded-state/unique-token/UnderlyingAsset.js

```diff
@@ -87,13 +95,9 @@ export default function UnderlyingAsset({
             height={30}
             marginLeft={6}
           >
-            <Column
-              height={16}
-              marginTop={android ? 8 : 3}
-              width={percentageAllocation * 2}
-            >
+            <Column height={16} marginTop={android ? 8 : 3} width={columnWidth}>
```

Comment:
I think we use checks for `ios` more often than Android. I'd flip it (but of course it doesn't really matter).

```suggestion
            <Column height={16} marginTop={ios ? 3 : 8} width={columnWidth}>
```

Reply:
I searched in the project and found out that android is mentioned 234 times and ios 122 times in conditional.
But I can replace it if it going to be a pattern

Reply:
Haha, wow. It's ok then. I didn't bother to check, only relied on intuition.